docker/cron/generate_sitemap/generate_sitemap.py

```diff
@@ -78,22 +77,8 @@ def generate_sitemap_for_ecosystem(ecosystem: str, base_url: str) -> None:
   tree.write(filename, encoding='utf-8', xml_declaration=True)
 
 
-def compress_file(file_path: str) -> str:
```

Comment:
Is there any particular reason you removed the compression?

Reply:
I'm not 100% sure how to get GCS to serve .gz files directly as their non compressed version (with a .xml extension) (and setting the content-type correctly).

And since the performance and storage cost is moved to GCS, it doesn't matter as much that there is more bandwidth being sent. And this is mostly here for search engines crawlers anyway, where latency is not as much of an issue.